configuration.md

```diff
@@ -555,6 +558,31 @@ IRONdb starts.
 
 Default: The number of physical processor cores in the system.
 
+#### surrogate_database activity_tracking
+
```

Comment:
This paragraph and the next one: "Directs IRONdb as to if it should" is... rather tortured grammar.  Elsewhere we've maintained an active voice, so perhaps "Maintain activity windows for metrics", and "Maintain latest values...".